packages/payment-experience-api/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM registry.access.redhat.com/ubi8/nodejs-14 as base
+
+# Install yarn
+RUN npm install -g yarn
+
+# Copy monorepo general contents
+COPY ./package*.json .
+COPY ./tsconfig.json ./tsconfig.json
+COPY ./tsconfig.base.json ./tsconfig.base.json
+COPY ./jest.config.js ./jest.config.js
+
+# Copy package and dependent packages
+COPY ./packages/core ./packages/core
+COPY ./packages/order-backend ./packages/order-backend
+COPY ./packages/price-backend ./packages/price-backend
```

Comment:
I wonder do we need these dependencies:
- price-backend
- product-backend
Could speed up building time if not required.